packages/tests/shadydom/event-path.html

```diff
@@ -930,6 +937,123 @@
       w.dispatchEvent(new Event('foo'));
       assert.isTrue(heardEvent);
     });
+
+    suite('unassigned slottable targets', () => {
```

Comment:
Suggest adding tests that stress this code path (if we don't have them) for:
1. events that bubble v. not
2. events that are composed v. not
3. events are not sent to removed event listeners

Reply:
Added. Also, good call, the listeners weren't being removed properly.